datus-clickzetta/datus_clickzetta/connector.py

```
@@ -878,4 +884,85 @@ def identifier(
+                return ExecuteSQLResult(
+                    columns=[],
+                    rows=rows,
+                    row_count=len(rows),
+                    execution_time_ms=0.0,
+                    sql_type=parse_sql_type(sql_query)
+                )
+            else:
+                # Default to CSV format
+                return self.execute_csv(sql_query)
+        except Exception as e:
+            logger.error(f"Error executing query: {e}")
```

Ruff (0.14.5)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)